rero_mef/views.py

```
@@ -15,15 +15,96 @@
+
+    actions = []
+    for target in targets:
+        try:
+            if current_search_client.indices.exists_alias(name=target):
+                mapping = current_search_client.indices.get_alias(name=target)
+                actions.extend(
+                    {"add": {"index": index_name, "alias": alias_name}}
+                    for index_name in mapping
+                )
+                continue
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)